contracts/interfaces/IFeeCollector.sol

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/// @title A smart contract for registering vaults for payments.
+interface IFeeCollector {
+    /// @notice An item that represents whom to transfer fees to and what percentage of the fees
+    /// should be sent (expressed in basis points).
+    struct FeeShare {
+        address payable treasury;
+        uint96 feeShareBps;
+    }
+
+    struct Vault {
+        address payable owner;
+        address token;
+        bool multiplePayments;
+        uint128 fee;
+        uint128 collected;
```

Comment:
I would maybe consider calling this `toCollect` based on its functionality, ie. when money is put into the vault it is increased, and when money is withdrawn/collected from the vault it goes to 0.

Or maybe if the goal is to describe that it is the amount that the vault is holding, naming it `currentVaultHoldings` or something like that.

It can just be confusing because one doesn't know at first whether the collecting is referring to those withdrawing aka they are "collecting" from the vault, or that the vault is the one "collecting" fee payments.

Reply:
My thought process was closed to your second suggestion. Never really thought about it more, but true, this might be ambiguous, I'll consider changing it to a better name